src/tasks/daily/daily_demo_mixin.py

```
@@ -135,20 +132,49 @@ def _demo_click_track_and_transfer(self):
+
+        if self.wait_until(level_changed, time_out=time_out, settle_time=0.2, raise_if_not_found=False):
+            return changed["level"]
+        return None
+
+    def _level_tip_box(self):
+        return self.box_of_screen(0.120, 0.724, 0.803, 0.750)
+
+    def _level_from_tip(self, result):
+        start_x = 0.125 #等级信息区域左边界占屏幕宽度的比例
+        end_x = 0.802 #等级信息区域右边界占屏幕宽度的比例
+        level_all = 11 #总共的等级数，从0级到10级
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)